config/migrations/20200903101700-procedurestap-in-new-model.sparql

```diff
@@ -0,0 +1,19 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX dbpedia:  <http://dbpedia.org/ontology/>
+
+DELETE{
+  GRAPH ?g {
+  ?ps a dbpedia:UnitOfWork .
+  }
+}
+INSERT {
+  GRAPH ?g {
+  ?ps a dossier:Procedurestap .
+  }
+} WHERE{
+  SELECT ?g, ?ps WHERE {
```

Comment:
Zie [Example 5](https://www.w3.org/TR/sparql11-update/#deleteInsert) voor de opbouw van een `DELETE` `INSERT` `WHERE` query (de `WITH` in het voorbeeld is een andere manier van de grafe specifieren en mag je vergeten). Hier heb je wat te veel in de `WHERE` clause :-)